test/functorch/test_vmap.py

```diff
@@ -650,24 +650,22 @@ def foo(x):
         vmap(foo, in_dims=(1,))(torch.randn(2, 3))
 
     def test_fallback_does_not_warn_by_default(self):
-        # NB: One day we will implement a batching rule for torch.atan2.
-        # If/when we do, this test should be replaced to test the fallback
-        # path on another operator to avoid bitrot.
-        op = torch.copysign
+        op = torch._test_functorch_fallback
         x = torch.randn(11)
         y = torch.randn(11)
         with warnings.catch_warnings(record=True) as wa:
             torch.vmap(op)(x, y)
             # The single warning here is the "vmap is experimental"
             # warning, not a warning from the vmap fallback path.
+            print(wa)
```

Comment:
What's the print for?

Reply:
Forgot to take it out after debugging, sorry!